docs/onboarding.rst

```diff
@@ -127,3 +127,47 @@ the file manager |Nautilus| and double-click on the public key to import it.
 
 .. |Nautilus| image:: images/nautilus.png
 .. |Importing Journalist GPG Keys| image:: images/install/importkey.png
+
+Verify Journalist Setup
+-----------------------
+
+Once the journalist device and account have been provisioned, then the
+administrator should run through the following steps with *each journalist* to
+verify the journalist is set up for SecureDrop.
+
+The journalist should verify that they:
+
+1. Have their own *Journalist Tails USB* that they have verified they are able
+   to boot on the *Journalist Workstation*.
+
+.. note:: It is important that they test on the same *Journalist Tails USB* and
+   the same *Journalist Workstation* they will be using on a day to day basis.
+   Issues may arise due to differences in USB drives or laptop models.
+
+2. Verify they are able to decrypt the persistent volume on the *Journalist
+   Tails Drive*.
```

Comment:
Keep the terminology consistent. *Journalist Tails Drive* != *Journalist Tails USB*. A quick grep shows that *Journalist Tails USB* is the preferred terminology:

```
securedrop ~> find ./docs -type f -name '*.rst' -exec grep 'Journalist Tails USB' '{}' \; | wc -l
       6
securedrop ~> find ./docs -type f -name '*.rst' -exec grep 'Journalist Tails Drive' '{}' \; | wc -l
       0
```